packages/vscode-mdx/src/extension.js

```diff
@@ -6,7 +6,7 @@
 import * as languageServerProtocol from '@volar/language-server/protocol.js'
 import {activateAutoInsertion, getTsdk, supportLabsVersion} from '@volar/vscode'
 import {languages, workspace} from 'vscode'
-import {LanguageClient, TransportKind} from 'vscode-languageclient/node.js'
+import {LanguageClient, TransportKind} from '@volar/vscode/node.js'
```

Comment:
Is there any particular reason to import these from `@volar/vscode/node.js`, not `@volar/vscode`? I’m mostly asking, because `@volar/vscode` is already imported 2 lines above.

Reply:
`@volar/vscode/node.js` is the re-export of `vscode-languageclient/node.js`, `@volar/vscode` is the re-export of `vscode-languageclient/index.js`, and LanguageClient, TransportKind Only exported in `vscode-languageclient/node.js`.

As context, `@volar/vscode` re-exports `vscode-languageclient` to ensure that the lsp modules used downstream are consistent with what `@volar/language-server` depends on, when `@volar/language-server` is in use` vscode-languageserver@9.0.1`, and problems may occur if `vscode-languageclient@8.x` is being used downstream.

Reply:
`@volar/vscode/node.js` is a re-export of `vscode-languageclient/node.js`, and `@volar/vscode` is a re-export of `vscode-languageclient/index.js`. The LanguageClient and TransportKind are only exported in `vscode-languageclient/node.js`.

In this context, the re-export of `@volar/vscode` from `vscode-languageclient` ensures that the LSP modules used downstream are consistent with those relied upon by `@volar/language-server`. This is crucial in cases where `@volar/language-server` is utilizing `vscode-languageserver@9.0.1`, and downstream is using `vscode-languageserver@8.x`, as this disparity may lead to issues.